src/one_dragon_qt/widgets/multi_selection_combo_box.py

```
@@ -0,0 +1,548 @@
+        if not emit_signal:
+            self.blockSignals(False)
+
+    def get_value(self) -> list:
+        """获取选中的值列表（实际值，不是显示文本）
+
+        Returns:
+            选中的实际值列表
+        """
+        return [item.value for item in self._selected_items]
+
+    def init_with_value(self, target_value: list = None) -> None:
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)